load-tests/stack_analysis.py

```diff
@@ -0,0 +1,54 @@
+"""Stack Analysis Load test."""
+
+import os
+import datetime
+import time
+from requests_futures.sessions import FuturesSession
+
+start_time = datetime.datetime.utcnow().strftime("%a %b %d %H:%M:%S %Z %Y")
+print("TEST START TIME: {}".format(start_time))
+
+
+three_scale_token = os.getenv('THREE_SCALE_PREVIEW_USER_KEY', '')
+api_url = os.getenv('F8A_API_V2_URL')
+api_suffix = ''
+
+
+def close_fps(fp_arr):
+    """Close all the file pointers."""
+    for file_p in fp_arr:
+        file_p.close()
+
+
+fp_array = []
+futures = []
+ecosystem_file_mapping = {'npm': 'npmlist.json', 'pypi': 'pylist.json', 'maven': 'dependencies.txt'}
+
+params = {'x-3scale-account-secret': three_scale_token}
```

Comment:
the parameter was user_key right ?

Reply:
That is if you are using the 3-scale API. I am directly hitting the API server.